ecommerce/OrderController.php

```diff
@@ -532,9 +532,17 @@ public function make_refund() {
 
 			$order_data->payment_status = $update_data['payment_status'];
 			$order_data->order_status   = $update_data['order_status'];
-			do_action( 'tutor_after_order_refund', $order_data, $amount );
+			do_action( 'tutor_after_order_refund', $order_id, $amount, $reason );
+
+			$res_msg = __( 'Order refund successful', 'tutor' );
 
-			$this->json_response( __( 'Order refund successful', 'tutor' ) );
+			try {
+				$this->refund_from_payment_gateway( $order_id, $amount, $reason );
```

Comment:
Why we are not using `tutor_after_order_refund` hook to process refund from payment gateway?